src/ybadge.h

```diff
@@ -31,6 +31,7 @@ void leds_set_color(uint16_t index, uint8_t red, uint8_t green, uint8_t blue);
 ////////////////////////////// Switches/Buttons ///////////////////////////////
 bool switches_get(uint8_t switch_idx);
 bool buttons_get(uint8_t button_idx);
+int knob_get();
```

Comment:
Do you know the range of values this can return?

Reply:
`analogRead` returns a value from 0 to 4095. I map those values to 0 to 100.